src/HIE/Bios/Cradle.hs

```diff
@@ -317,7 +317,7 @@ getCabalWrapperTool (ghcPath, ghcArgs) = do
             writeFile wrapper_hs cabalWrapperHs
             createDirectoryIfMissing True cacheDir
             let ghc = (proc ghcPath $ ghcArgs ++ ["-o", wrapper_fp, wrapper_hs])
-                        { cwd = Just (takeDirectory wrapper_hs) }
+                        { cwd = Just wdir }
```

Comment:
Out of curiosity: why te compilation is in the working dir if  the wrapper are gonna be as before?

Reply:
Because we are using stack now to compile wrapper, since some users might not have a raw ghc installed. But if you execute stack exec ghc in the appdata something directory, we are not picking up the current stack directory, but the global one. Thus, execute stack ghc in the cradle directory, to pick up the correct ghc version

Reply:
Does this produce `.hi` and `.o` files in the working directory? I would guess it's ok still.

Reply:
Good point, no idea, will test it.

Reply:
Compilation artifacts are in the same folder as the location of the input file. In this case, the compilation artifacts are found in %TMP%.